package.json

```diff
@@ -1,46 +1,30 @@
 {
-  "name": "mydatahelps-starterkit",
-  "version": "1.0.0",
+  "name": "mydatahelpsstarterkit",
   "private": true,
-  "dependencies": {
-    "@careevolution/mydatahelps-js": "^3.2.0",
-    "@careevolution/mydatahelps-ui": "^1.3.7",
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^17.0.21",
-    "@types/react": "^17.0.40",
-    "@types/react-dom": "^17.0.13",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^6.2.1",
```

Comment:
`react-router-dom` needs to still be a dependency.  On a fresh checkout, after an `npm install` you end up with an error in `main.tsx` that `react-router-dom` is an unresolved dependency.

Reply:
Thanks for pointing this out. I was not getting the error locally, and re added the dependancy